src/components/Expander/Expander.jsx

```diff
@@ -102,6 +106,44 @@ const Expander = createClass({
 		}
 	},
 
+	storeRef(name) {
+		return (ref) => {
+			this.Refs[name] = ref;
+		};
+	},
+
+	componentWillMount() {
+		this.Refs = {};
+		this.isAnimated = false;
+		this._labelKey = 0;
+	},
+
+	componentDidMount() {
+		_.delay(() => {
+			this.setState({
+				maxHeight: this.Refs.contentInner.offsetHeight,
+			});
+			this.isAnimated = true;
+		}, 32);
```

Comment:
What's the purpose of this constant? Could probably move it to a `const` to make a little clearer